packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -259,6 +260,23 @@ abstract contract FundingRateApplier is FeePayer {
         return fundingRate.rate;
     }
 
+    function _validateFundingRate(FixedPoint.Signed memory rate) internal view {
+        // Constraining the range of funding rates limits the PfC for any dishonest proposer and enhances the
```

Comment:
nit: do you want to put this comment above the function name?

Reply:
done